packages/mcfunction/test/parser/utils.ts

```diff
@@ -0,0 +1,34 @@
+import type { RootTreeNode } from '../../../java-edition/lib/dependency/mcmeta.js'
```

Comment:
The `mcfunction` package also has a `RootTreeNode` type, you should probably use that one.

Reply:
that one leads to type errors.

![image](https://github.com/SpyglassMC/Spyglass/assets/13565346/a9e4dac9-fe3f-482d-a4ec-12dd4affdcf7)

![image](https://github.com/SpyglassMC/Spyglass/assets/13565346/90a41c23-af45-4396-9fc8-78cfc28d626a)

Reply:
Okay I guess that is caused by using the `initialize` function from the `java-edition` package in a `mcfunction` test. Which feels wrong, as discussed in the other comment.